lib/inferno/entities/test_suite.rb

```diff
@@ -186,6 +187,25 @@ def suite_summary(suite_summary = nil)
 
           @suite_summary = format_markdown(suite_summary)
         end
+
+        # Get the TestKit this suite belongs to
+        #
+        # @return [Inferno::Entities::TestKit]
+        def test_kit
+          return @test_kit if @test_kit
+
+          module_name = name
+
+          while module_name.present? && @test_kit.nil?
```

Comment:
Just out of curiosity, is there a reason this is a loop instead of just a conditional?

Reply:
Yes, we need to keep going up the namespaces until we find the one with the metadata. US Core for example has nested namespaces: `USCoreTestKit::USCoreV311::USCoreTestSuite`.